src/docgen/transpile/typescript.ts

```diff
@@ -236,6 +238,11 @@ export class TypeScriptTranspile extends transpile.TranspileBase {
     const tf = transpiled.typeReference.toString({
       typeFormatter: (t) => t.name,
     });
+
+    if (transpiled.variadic) {
+      return `${transpiled.name}${transpiled.optional ? '?' : ''}: ...${tf}[]`;
```

Comment:
```suggestion
      return `${transpiled.name}${transpiled.optional ? '?' : ''}: ...${this.listOf(tf)}`;
```

Think about whether this is reusable enough. As you mentioned, this PR is still missing the variadic formatting in the `Type` section of the metadata. This probably means you'll have to repeat this logic somewhere else - how can we make sure we only define this formatting once?